types/recharts/index.d.ts

```diff
@@ -40,7 +41,7 @@ export interface AreaProps {
 	name?: string | number;
 	isAnimationActive?: boolean;
 	animationBegin?: number;
-	animationDuration?: number;
+	// animationDuration?: number;
```

Comment:
Ok, removed the lines. I was not sure if I should remove them or change them, that they matches the definitions from CSSStyleDeclaration...